packages/insight-previous/src/providers/bwc/bwc.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+
+import * as BWC from 'bitcore-wallet-client';
```

Comment:
Why is BWC needed in insight?

Reply:
Is because the search bar checks if you entered a valid address before making requests. But, you're right, it's not necessary to use BWC, I have to use bitcore-lib and bitcore-lib-cash instead. Thanks!